getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1464,62 +1464,104 @@ freed.
 Signals
 ~~~~~~~
 
-It is often desired to send a notification that something happened in an
-instance. GDScript supports creation of built-in Godot signals.
-Declaring a signal in GDScript is easy using the `signal` keyword.
+Signals are a way to send notification messages from an instance that
+other instances can listen to in a generic way. Create custom signals for
+a class using the ``signal`` keyword.
 
 ::
 
-    # No arguments.
-    signal your_signal_name
-    # With arguments.
-    signal your_signal_name_with_args(a, b)
+   # Signal with no arguments
+   signal your_signal_name
 
-These signals can be connected in the editor or from code like regular signals.
-Take the instance of a class where the signal was
-declared and connect it to the method of another instance:
+   # Signal with two arguments
+   signal your_signal_name_with_args(a, b)
+
+These signals are connected to methods like regular signals in built-in
```

Comment:
These signals may be connected to methods in the same manner as you connect built-in signals of nodes such as :ref:`class_Button` or :ref:`class_RigidBody`.

Reply:
Changed to this in fb64a084cb334abad8e4a331704f183ddeba73bb